### ISSUE TITLE
关于epoch的设置

### ISSUE THREAD

**Issue by YuyangWei** [alexhw15/HiRe#3]
您好，感谢分享。我在论文中看到您设置的epoch是30000。但是，我将epoch设置为30000时，结果相差很大。能具体说明你大约在多少epoch得到最优参数吗？期刊您的回复。

**Comment by alexhw15** [comment]
I think the best epoch for nell 5-shot/3-shot/1-shot are 30000/40000/50000-ish, respectively.

**Comment by YuyangWei** [comment]
Thanks for your reply. I run the code with 1-shot setting command presented in Readme. But, the results significantly differ from the results reported in the paper. Can you tell me how to set a random seed or provide other processing techniques?

**Comment by alexhw15** [comment]
Hi YuyangWei,

I have re-run the code on another device using the codes, datasets and run/test scripts directly downloaded from this repo. The best epoch under 1-shot setting on Nell dataset is 53, 000. On validation set, the MRR and Hits@10/5/1 is 0.154, 0.328, 0.237 and 0.066. On test set (which related work and our paper reports on), the results are 0.286, 0.468, 0.398 and 0.188, consistent with what we've reported. Hope this is helpful to you.

**Comment by lssshuang** [comment]
我运行的结果在Nell 数据集上 1-shot 设置下的最佳 epoch 为 35, 000

**Comment by YuyangWei** [comment]
谢谢您的回复，我已经运行出和论文报告相近的结果

**Comment by lssshuang** [comment]
hello, I run the code 1-shot on WiKI dataset. But,the result of MRR is only 0.240 . Can you tell me the command to train the wiki dataset and the the best epoch for wiki?
